modules/couchdb/metrics.go

```diff
@@ -10,7 +12,8 @@ type cdbMetrics struct {
 	// https://docs.couchdb.org/en/stable/api/server/common.html#node-node-name-system
 	NodeSystem *cdbNodeSystem
 	// https://docs.couchdb.org/en/stable/api/database/common.html
-	DBStats map[string]*cdbDBStats
+	DBStats      map[string]*cdbDBStats
+	DBStatsMutex sync.Mutex
```

Comment:
doesnt belong here, just create the lock locally where you need it

Reply:
(this would remove all lint errors)

Reply:
Fixed